ddgr

```diff
@@ -1454,6 +1461,10 @@ class DdgCmd(object):
                 elif cmd == 'x':
                     Result.urlexpand = not Result.urlexpand
                     printerr('url expansion toggled.')
+                elif cmd[0] == 'c' and cmd[2:].isdigit():
+                    url = self._urltable[cmd[2:]]
```

Comment:
Add the current logic within exception handler and in case of a module not found exception, show the msg pyperclip is not installed.

Mention pyperclip in the README (dependencies section) as an optional dependency.

Reply:
I think it's also a good idea to import pyperclip within the exception handler locally at this point.